### ISSUE TITLE
Switch to using bumpver

### ISSUE THREAD

**Issue by agjohnson** [readthedocs/sphinx_rtd_theme#1324]
We switched out bumpversion/bump2version for bumpver on readthedocs.org and other repos. Probably best to also switch this repo. The tooling is very similar.

- [ ] Change configuration
- [ ] Doc updates for release procedures

**Comment by agjohnson** [comment]
Also, noted in a chat, we'll eventually have a slightly altered version number in package.json, which is primarily informational anyways, though might affect package installation too. This would be a good time to attempt reconfiguring package.json, as we are switching tools.

We don't really need to worry about the package.json version until Node 18 it sounds though, and are stuck with Node 14 for the forseeable future.

**Comment by benjaoming** [comment]
This sounds like a really good idea. Especially to sort out the different version schemes of PEP440 and npm, so the release process doesn't become vague or confusing.

I would really want to fix this, coinciding with #1362

**Comment by benjaoming** [comment]
Since package.json isn't part of the release but only part of the build stack, I think that truncating the version part (if possible) would be a great way to handle the problem :+1:

**Comment by agjohnson** [comment]
Let's just leave the version scheme as it is. It's not important to have this match Node versioning patterns, as we don't release to NPM. It's just updated in package.json for consistency.

**Comment by benjaoming** [comment]
Yes, [I agree](https://github.com/readthedocs/sphinx_rtd_theme/issues/1324#issuecomment-1279136005) :+1:

**Comment by agjohnson** [comment]
Yup, I read that. You still seem to be advocating for altering the version for package.json. I'm describing take *no* action for package.json version -- it should match the version string from all other files.

**Comment by benjaoming** [comment]
Since we don't distribute this, I am happy to do whatever is most feasible to implement, as long as our npm commands "eat" it :)

**Comment by humitos** [comment]
We have the release process documented at https://sphinx-rtd-theme.readthedocs.io/en/stable/contributing.html#releasing-the-theme and we are already using bump2version. Closing.

**Comment by agjohnson** [comment]
This issue is low priority, but this issue is still valid. This is the only repo we're using bump2version, all other repos use bumpver. I think it still makes sense to standardize and it probably is not much work really.

**Comment by humitos** [comment]
Ahh.... I miss-read it. OK, re-opening then 😄
